app/PaymentDrivers/Stripe/ACH.php

```diff
@@ -168,14 +168,12 @@ public function paymentView(array $data)
         $data['amount'] = $this->stripe->convertToStripeAmount($data['total']['amount_with_fee'], $this->stripe->client->currency()->precision, $this->stripe->client->currency());
         $amount = $data['total']['amount_with_fee'];
 
-        $invoice = Invoice::whereIn('id', $this->transformKeys(array_column($this->stripe->payment_hash->invoices(), 'invoice_id')))
-                          ->withTrashed()
-                          ->first();
+        $invoice_numbers = collect($data['invoices'])->pluck('invoice_number')
 
-        if ($invoice) {
-            $description = ctrans('texts.stripe_paymenttext', ['invoicenumber' => $invoice->number, 'amount' => Number::formatMoney($amount, $this->stripe->client), 'client' => $this->stripe->client->present()->name()]);
+        if ($invoice_numbers.length > 0) {
```

Comment:
.length is javascript :)

perhaps you'll want to check strlen > 0 ?

Reply:
Resolved